tools/azure-sdk-tools/devtools_testutils/sanitizers.py

```diff
@@ -0,0 +1,199 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import requests
+from typing import TYPE_CHECKING
+
+from .config import PROXY_URL
+
+if TYPE_CHECKING:
+    from typing import Any, Dict
+
+
+def add_body_key_sanitizer(**kwargs):
+    # type: (**Any) -> None
+    """Registers a sanitizer that offers regex update of a specific JTokenPath within a returned body.
+
+    For example, "TableName" within a json response body having its value replaced by whatever substitution is offered.
+
+    :keyword str json_path: The SelectToken path (which could possibly match multiple entries) that will be used to
+        select JTokens for value replacement.
+    :keyword str value: The substitution value.
+    :keyword str regex: A regex. Can be defined as a simple regex replace OR if groupForReplace is set, a subsitution
+        operation. Defaults to replacing the entire string.
+    :keyword str group_for_replace: The capture group that needs to be operated upon. Do not provide if you're invoking
+        a simple replacement operation.
+    """
+
+    request_args = {}
+    request_args["jsonPath"] = kwargs.get("json_path")
+    request_args["value"] = kwargs.get("value")
+    request_args["regex"] = kwargs.get("regex")
+    request_args["groupForReplace"] = kwargs.get("group_for_replace")
+
+    _send_request("BodyKeySanitizer", request_args)
+
+
+def add_body_regex_sanitizer(**kwargs):
+    # type: (**Any) -> None
+    """Registers a sanitizer that offers regex replace within a returned body.
+    
+    Specifically, this means regex applying to the raw JSON. If you are attempting to simply replace a specific key, the
+    BodyKeySanitizer is probably the way to go.
+
+    :keyword str value: The substitution value.
+    :keyword str regex: A regex. Can be defined as a simple regex, or if a ``group_for_replace`` is provided, a
+        substitution operation.
+    :keyword str group_for_replace: The capture group that needs to be operated upon. Do not provide if you're invoking
+        a simple replacement operation.
+    """
+
+    request_args = {}
+    request_args["value"] = kwargs.get("value")
+    request_args["regex"] = kwargs.get("regex")
+    request_args["groupForReplace"] = kwargs.get("group_for_replace")
```

Comment:
There are some similar lines in constructing request_args, is it possible to do some refactor based on this point? For example, have one method for preparing request arguments.